src/components/utils/format.js

```diff
@@ -31,9 +31,11 @@ const FormatDate = ({
   )
 }
 
-const FormatNumber = ({ number, nullValue = false }) => (
-  <>{number || number === 0 ? number.toLocaleString() : nullValue || 'N/A'}</>
-)
+const FormatNumber = ({ number, nullValue = false }) => {
```

Comment:
@schwartzadev what if we make the option for precision a prop for `FormatNumber`?

Reply:
Good call!